RoaringBitmap/src/main/java/org/roaringbitmap/buffer/RunBatchIterator.java

```diff
@@ -62,14 +62,15 @@ public void advanceIfNeeded(char target) {
     do {
       int runStart = runs.getValue(run);
       int runLength = runs.getLength(run);
-      if (runStart <= target && runStart + runLength > target) {
-        cursor = target - runStart;
-        break;
-      }
+      int offset = target - runStart;
```

Comment:
Could you bring this declaration down to below line 69 so it's closer to where it's used please?

Reply:
> Could you bring this declaration down to below line 69 so it's closer to where it's used please?

:white_check_mark: